### ISSUE TITLE
Message tag field name discrepancy?

### ISSUE THREAD

**Issue by pascoemitch** [jgorset/facebook-messenger#261]
I was just looking through the documentation here and it specifies `message_type` and noticed that this field is named `messaging_type` in [Facebook's documentation](https://developers.facebook.com/docs/messenger-platform/reference/send-api#properties).

It looks like `Bot.deliver` just passes the whole message hash through to Facebook so should this be updated or am I missing something?

```ruby
# Docs

Bot.deliver({
  recipient: {
    id: '45123'
  },
  message: {
    text: 'Human?'
  },
  message_type: Facebook::Messenger::Bot::MessagingType::MESSAGE_TAG
  tag: Facebook::Messenger::Bot::Tag::NON_PROMOTIONAL_SUBSCRIPTION
}, access_token: ENV['ACCESS_TOKEN'])

```

```ruby
# Facebook::Messenger::Bot

def deliver(message, access_token:, app_secret_proof: nil)
  query = {
    access_token: access_token
  }
  query[:appsecret_proof] = app_secret_proof if app_secret_proof
  response = post '/messages',
                  body: JSON.dump(message),
                  format: :json,
                  query: query

  Facebook::Messenger::Bot::ErrorParser.raise_errors_from(response)

  response.body
end
```

Thanks

**Comment by jgorset** [comment]
Yeah! That's exactly right. Good catch, thanks @pascoemitch!

**Comment by jgorset** [comment]
Fixed in bb8d31e ✨
